org.gtk.oomox.appdata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.gtk.oomox</id>
+  <metadata_license>GPLv3</metadata_license>
```

Comment:
If you wrote this `CC0-1.0` is preferred.

Reply:
you mean what that's a license of flatpak xml and json files, not of application itself?

Reply:
Yes - of the metadata.